solutions/agents/version.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  # module uses nullable feature which is only available in versions >= 1.1.0
+  required_version = ">= 1.1.0, <1.7.0"
+
```

Comment:
Can you leave a code comment to say `# Lock DA into an exact provider versions - renovate automation will keep it updated`

Reply:
done